libafl/src/events/tcp.rs

```diff
@@ -711,6 +689,14 @@ where
     EMH: EventManagerHooksTuple<S>,
     S: State,
 {
+    fn should_send(&self) -> bool {
+        if let Some(throttle) = self.throttle {
+            libafl_bolts::current_time() - self.last_sent > throttle
```

Comment:
Looks like `self.last_sent` is never set(?)

Reply:
nice catch 😉